plugins/inventory/proxmox.py

```diff
@@ -268,7 +268,9 @@ def _get_node_ip(self, node):
             try:
```

Comment:
I would actually change this to: 
It's possible for a proxmox node to have a combination:
* single public and/or multiple private
* multiple public and multiple private
* multiple private
```
    def _get_node_ip(self, node):
        ret = self._get_json("%s/api2/json/nodes/%s/network" % (self.proxmox_url, node))

        # first try to get the first available public IP if it exists
        for iface in ret:
            addr = iface.get('address', None)
            if all([addr != None, ipaddress.ip_address(addr).is_global]):
                return addr

        # get the first available private IP in the list if no public ones are available, 
        # excluding loopback addresses
        for iface in ret:
            addr = iface.get('address', None)
            if all([addr != None, not ipaddress.ip_address(addr).is_loopback, 
                    ipaddress.ip_address(addr).is_private]):
                return addr

        # no IP address found
        return None
```